logstash-core/src/test/resources/org/logstash/config/ir/cache/pipeline_reuse_baseline.conf

```diff
@@ -25,7 +26,8 @@ filter {
 }
 output {
   if [a] { noop {} }
+  else { noop {} }
   stdout {}
   stdout {}
-  stdout {}
+#  stdout {}
```

Comment:
We need to remove else the datasource that invokes all the outputs has 5 child datasources (1 for the if, 1 for the complement and 3 for the stdout filters, out for each invocation). This would generate a Datasource with 5 childs, while in the `pipeline_reuse_test.conf` output section the child are 4, (1 for the first `if`statement, 1 for the complement of the first if, 1 for the second `if`statement, 1 for the complement of the second if).